src/faker-factory.ts

```diff
@@ -1,94 +1,94 @@
     case 'de_AT':
-      return import('faker/locale/de_AT');
+      return import('@faker-js/faker/locale/de_AT');
     case 'de_CH':
-      return import('faker/locale/de_CH');
+      return import('@faker-js/faker/locale/de_CH');
     case 'en':
-      return import('faker/locale/en');
+      return import('@faker-js/faker/locale/en');
     case 'en_AU':
-      return import('faker/locale/en_AU');
+      return import('@faker-js/faker/locale/en_AU');
     case 'en_AU_ocker':
-      return import('faker/locale/en_AU_ocker');
+      return import('@faker-js/faker/locale/en_AU_ocker');
     case 'en_BORK':
-      return import('faker/locale/en_BORK');
+      return import('@faker-js/faker/locale/en_BORK');
     case 'en_CA':
-      return import('faker/locale/en_CA');
+      return import('@faker-js/faker/locale/en_CA');
     case 'en_GB':
-      return import('faker/locale/en_GB');
+      return import('@faker-js/faker/locale/en_GB');
     case 'en_IE':
-      return import('faker/locale/en_IE');
+      return import('@faker-js/faker/locale/en_IE');
     case 'en_IND':
-      return import('faker/locale/en_IND');
+      return import('@faker-js/faker/locale/en_IND');
     case 'en_US':
-      return import('faker/locale/en_US');
+      return import('@faker-js/faker/locale/en_US');
     case 'en_ZA':
-      return import('faker/locale/en_ZA');
+      return import('@faker-js/faker/locale/en_ZA');
     case 'es':
-      return import('faker/locale/es');
+      return import('@faker-js/faker/locale/es');
     case 'es_MX':
-      return import('faker/locale/es_MX');
+      return import('@faker-js/faker/locale/es_MX');
     case 'fa':
-      return import('faker/locale/fa');
+      return import('@faker-js/faker/locale/fa');
     case 'fi':
-      return import('faker/locale/fi');
+      return import('@faker-js/faker/locale/fi');
     case 'fr':
-      return import('faker/locale/fr');
+      return import('@faker-js/faker/locale/fr');
     case 'fr_CA':
-      return import('faker/locale/fr_CA');
+      return import('@faker-js/faker/locale/fr_CA');
     case 'fr_CH':
-      return import('faker/locale/fr_CH');
+      return import('@faker-js/faker/locale/fr_CH');
     case 'ge':
-      return import('faker/locale/ge');
+      return import('@faker-js/faker/locale/ge');
     case 'id_ID':
-      return import('faker/locale/id_ID');
+      return import('@faker-js/faker/locale/id_ID');
     case 'it':
-      return import('faker/locale/it');
+      return import('@faker-js/faker/locale/it');
     case 'ja':
-      return import('faker/locale/ja');
+      return import('@faker-js/faker/locale/ja');
     case 'ko':
-      return import('faker/locale/ko');
+      return import('@faker-js/faker/locale/ko');
     case 'nb_NO':
-      return import('faker/locale/nb_NO');
-    case 'nep':
-      return import('faker/locale/nep');
+      return import('@faker-js/faker/locale/nb_NO');
+    case 'ne':
+      return import('@faker-js/faker/locale/ne');
```

Comment:
it's intended to replace `nep` with `ne` based on https://fakerjs.dev/api/localization.html#localization